charts/reform-scan-blob-router/values.yaml

```diff
@@ -44,6 +44,7 @@ java:
 
     SMTP_HOST: "false"
     SEND_DAILY_REPORT_ENABLED: "false"
+    RECONCILIATION_ENABLED: "true"
```

Comment:
may be safer side let's keep it disabled and only enable in aat template ?
Once auth story is played this will be removed completely so you won't need to bother with flux PR's

Reply:
Updated the templates